docs/word/word-add-ins-events.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Work with events using the Word JavaScript API
+description: A list of events for Word JavaScript objects. This includes information on using event handlers and the associated patterns.
+ms.date: 07/09/2024
+ms.localizationpriority: medium
+---
+
+# Work with events using the Word JavaScript API
+
+This article describes important concepts related to working with events in Word and provides code samples that show how to register event handlers, handle events, and remove event handlers using the Word JavaScript API.
+
+## Events in Word
+
+Each time certain types of changes occur in a Word document, an event notification fires. By using the Word JavaScript API, you can register event handlers that allow your add-in to automatically run a designated function when a specific event occurs. The following events are currently supported.
```

Comment:
I know the pluralization was correct, but I think simplifying the sentences a bit would help speed readers.

```suggestion
When certain changes occur in a Word document, event notifications fire. The Word JavaScript APIs let you register event handlers that allow your add-in to automatically run designated functions when those changes occur. The following events are currently supported.
```